salsa_dancing_molecules/variables.py

```diff
@@ -15,7 +14,6 @@ class Variables:
     kinetic_energies = []
     atomic_pressures = []
     temperatures = []
-    specific_heat_NVE = []

```

Comment:
Could you please explain in the commit message why you are removing this from our output file? It is not obvious to me.